internal/cli/kraft/cloud/deploy/deploy.go

```diff
@@ -64,10 +64,15 @@ func NewCmd() *cobra.Command {
 	cmd, err := cmdfactory.New(&DeployOptions{}, cobra.Command{
 		Short:   "Deploy your application",
 		Use:     "deploy",
-		Aliases: []string{"launch", "run"},
+		Aliases: []string{"depl"},
```

Comment:
I use these `run` too much, can we please just add `depl`?

Reply:
reverted